packages/mux-player/src/media-chrome/time-display.ts

```diff
@@ -60,6 +62,23 @@ class MxpTimeDisplay extends HTMLElement {
 
     this.addEventListener('click', this.toggleTimeDisplay);
   }
+
+  attributeChangedCallback(attrName: string, _oldValue: string | null, newValue: string | null) {
+    if (attrName === 'hide-duration') {
+      if (typeof newValue === 'string') {
```

Comment:
is there a particular reason this check is based on typeof? I thought we've been using `newValue == null` to see whether it was true or false for a boolean attribute

Reply:
nope. Hadn't noticed we'd settled on a soft rule/convention for attributeChangedCallback, but I'll change it for consistency.